packages/contracts/deploy/015-UpgradeGovernor.ts

```diff
@@ -41,12 +41,16 @@ const deployFn: DeployFunction = async (hre) => {
 
   // Check variable
   assert((await governor.timelock()) === timeLockProxyAddress)
-  assert((await governor.votingDelay()) === hre.deployConfig.votingDelayBlocks)
   assert(
-    (await governor.votingPeriod()) === hre.deployConfig.votingPeriodBlocks
+    (await governor.votingDelay()).toNumber() ===
```

Comment:
We don't need to check `securityCouncilTokenProxyAddress` and `votesQuorumFractionPercent`?

Reply:
The current contract structure does not provide getters for either element, so we have not made any modifications to the contract for this check, as it would require modifying the contract. 
Even if it is excluded from the validation, the validity of the token address is validated through the constructor logic, and we believe that the votesQuromFractionPercent will be valid along the same lines if the other values are validated successfully. 

Translated with www.DeepL.com/Translator (free version)

Reply:
I see. thanks!